spec/models/comment_spec.rb

```diff
@@ -184,11 +184,20 @@
         expect(comment.processed_html.include?("Hello <a")).to be(true)
       end
 
-      it "shortens long urls" do
-        comment.body_markdown = "Hello https://longurl.com/#{'x' * 100}?#{'y' * 100}"
+      it "shortens long urls does not strips out only urls" do
```

Comment:
Tiny change to group all failures:

```suggestion
      it "shortens long urls without removing formatting", :aggregate_failures do
```

Reply:
Whoops, missed the wording error there. Thanks for the review!